src/external-scripts/inkeep.ts

```diff
@@ -149,12 +205,52 @@ export const inkeepOnLoad = (apiKey: string) => {
   const baseSettings = {
     apiKey,
     transformSource,
+    theme: {
+      styles: [
```

Comment:
is the styles same for `website` and `voltaire`? Thinking to add it on `ably-ui` instead?

Reply:
Yeah, we're planning on adding an shared place for our JS, so once that's decided we'll update the repos